content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -164,8 +164,6 @@ Use the below command to replace or update the secret:
    ```bash
    cd dell-csi-helm-installer && wget -O myvalues.yaml https://github.com/dell/helm-charts/raw/csi-vxflexos-2.9.0/charts/csi-vxflexos/values.yaml
    ```  
-   >Note: To connect to a PowerFlex 4.5 array, edit the powerflexSdc parameter in your values.yaml file to use dellemc/sdc:4.5:  
```

Comment:
Removing this note since, starting from CSM 1.9 and v2.9.0 of the driver, SDC will be present in the sample file by default. For CSM 1.8 and v2.8.0 of the driver, it will be located under the v2 directory.